west.yml

```diff
@@ -303,7 +303,7 @@ manifest:
       groups:
         - crypto
     - name: mcuboot
-      revision: a2bc982b3379d51fefda3e17a6a067342dce1a8b
+      revision: 95773edf33082a1689795e0278c14b2da1e6ab74
```

Comment:
replace commit with commit from https://github.com/nordicjm/zephyr/tree/mcubootcherrypick

Reply:
Thank you for your suggestion.
I have extracted your patch, applied it, and pushed the changes.
Could you take a look?